src/migrators/cassandra/201606031548_init.clj

```diff
@@ -0,0 +1,56 @@
+(ns migrators.cassandra.201606031548-init
```

Comment:
we'll have to be careful of the ordering of these files when we have things coming in from multiple branches. Possibly a comment in the developer guidelines?